templates/errors/no-cookie.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'layouts/_base.html' %}
+
+{% set survey_title = _("Session expired") ~ " - " ~ _("Census 2021") %}
+
+{% block main %}
+  <h1>{{ _("Sorry there is a problem") }}</h1>
+  <p>{{ _("To access this page you need to <a href='{url}'>enter your 16-character access code</a>.").format(url = account_service_url) }}</p>
```

Comment:
`account_service_url` isn't set.

Reply:
Is there a fallback url I am able to use for this or does the content need to be changed?

Reply:
There isn't yet, but there probably should be. The difficulty will be we can only have one url, which will cause issues if they have tried to access the NI census. We shouldn't direct them to the wrong access code page.

Reply:
I've added the fallback URL now